docs/index.md

```diff
@@ -1140,6 +1140,31 @@ locator.registerInstance('browserifyPlugin', {
 * [catberry-debugger](https://github.com/catberry/catberry-debugger) – Chrome Dev Tools extension
 * [catberry-idea-plugin](https://github.com/catberry/catberry-idea-plugin) – Intellij platform plugin
 
+## Live Reloading
+
+Whilst Catberry doesn't have a built in live-reloading mechanism, it trivial to add it to your project by following these steps:
+
+### Server side live reloading
+
+* `npm install -g nodemon`
+* Change `package.json` file to run `nodaemon` instead of `node` by changing
+	* `debug` command to: `node ./build.js & nodemon --watch server --watch build.js --watch routes.js ./server.js`
+	* `debug-win` command to: `powershell -Command \"Start-Process -NoNewWindow nodemon ./build.js; Start-Process -NoNewWindow -Wait  -ArgumentList '--watch server --watch build.js --watch routes.js  ./server.js' nodemon\"`
+	
+### Client side live reloading
+* `npm install -S express-livereload`
+* Modify `server.js` and place the following code below `const app = express();`:
+```
+var livereload = require('express-livereload');
+livereload(app);
+```  
+* Add the following line to `head.hbs`:
```

Comment:
I would say "to the head component's template". We have to be template-agnostic and don't mention any particular template engine.